ajaanc/src/ancillarylist.cpp

```diff
@@ -986,6 +986,9 @@ AJAStatus AJAAncillaryList::AddFromDeviceAncBuffer (const NTV2Buffer & inAncBuff
 	const uint32_t	origPktCount	(outPackets.CountAncillaryData());
 	AJAStatus		result			(AJA_STATUS_SUCCESS);
 
+	if (inAncBuffer.GetByteCount() == 0)
```

Comment:
@janbrothanek This check isn't necessary. If `inAncBuffer` is empty/null, `BufferHasGUMPData` will return false (without crashing, it first checks for null/empty), and in the RTP 'else' case, `ancBuffer` will be empty/null too, so the `while` loop cannot be entered. The function will log zero packets added, and return `AJA_STATUS_SUCCESS`. I believe this is tested here: https://github.com/aja-video/libajantv2/blob/c6c784a9bd8fd8f27bc3421aec9ed52dd01a7dcc/ajaanc/test/main.cpp#L2352